.github/workflows/publish.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish Gem
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Publish Gem
+        uses: cadwallion/publish-rubygems-action@master
```

Comment:
I don't want to user a master version of a 3rd party GH action, especially when credentials are involved. The owner is a GH employee so they must have some special way of publishing because they don't have an `action.yml`.

I guess it sounds like I think other folks' GH actions are suspect.  Maybe I do. It's easy for us to _forget_ that we have this dependency when someone could be just harvesting our gem api keys.

It's so simple I'd rather just do it ourselves so we can never have to think about the security implications (well... depending on where we place the files in the action!)

Reply:
Updated to essentially do what the previous action did but do it all directly in the action from this repo.